scripts/install-mender.sh

```diff
@@ -326,7 +326,109 @@ do_install_missing_monitor_dirs () {
     fi
 }
 
+command_exists() {
+    command -v "$@" > /dev/null 2>&1
+}
+
+# Set the LSB_DIST and DIST_VERSION variables guessing the distribution and version;
+# It also checks if this is a forked Linux distro.
+# Credits: https://get.docker.com/
+check_dist_and_version() {
+    # Every system that we officially support has /etc/os-release
+    if [ -r /etc/os-release ]; then
+        LSB_DIST="$(. /etc/os-release && echo "$ID" | tr '[:upper:]' '[:lower:]')"
+    fi
+    case "$LSB_DIST" in
+        ubuntu)
+            if command_exists lsb_release; then
+                DIST_VERSION="$(lsb_release --codename | cut -f2)"
+            fi
+            if [ -z "$DIST_VERSION" ] && [ -r /etc/lsb-release ]; then
+                DIST_VERSION="$(. /etc/lsb-release && echo "$DISTRIB_CODENAME")"
+            fi
+            case "$DIST_VERSION" in
+                focal)
+                    DIST_VERSION="focal"
+                ;;
+                bionic)
+                    DIST_VERSION="bionic"
+                ;;
+                *)
+                    echo "ERROR: your distribution's version ($DIST_VERSION) is either not recognized or not supported."
+                    echo "Aborting."
+                    exit 1
+                ;;
+            esac
+        ;;
+        debian|raspbian)
+            DIST_VERSION="$(sed 's/\/.*//' /etc/debian_version | sed 's/\..*//')"
+            case "$DIST_VERSION" in
+                11)
+                    DIST_VERSION="bullseye"
+                ;;
+                10)
+                    DIST_VERSION="buster"
+                ;;
+                *)
+                    echo "ERROR: your distribution's version ($DIST_VERSION) is either not recognized or not supported."
+                    echo "Aborting."
+                    exit 1
+                ;;
+            esac
+        ;;
+        *)
+            echo "ERROR: your distribution ($LSB_DIST) is either not recognized or not supported."
+            echo "Aborting."
+            exit 1
+        ;;
+    esac
+
+    # Check for lsb_release command existence, it usually exists in forked distros
+    if command_exists lsb_release; then
+        # Check if the `-u` option is supported
+        set +e
+        lsb_release -a -u > /dev/null 2>&1
+        lsb_release_exit_code=$?
+        set -e
+
+        # Check if the command has exited successfully, it means we're in a forked distro
+        if [ "$lsb_release_exit_code" = "0" ]; then
+            # Get the upstream release info
+            LSB_DIST=$(lsb_release -a -u 2>&1 | tr '[:upper:]' '[:lower:]' | grep -E 'id' | cut -d ':' -f 2 | tr -d '[:space:]')
```

Comment:
I am getting:
```
lsb_release: error: no such option: -u
```

Reply:
As the comment says, it is available on forked distros only.
That's why we try the command, and then check the exit code.